js/vmouse.js

```diff
@@ -25,9 +25,13 @@ define( [ "jquery" ], function( jQuery ) {
 var dataPropertyName = "virtualMouseBindings",
 	touchTargetPropertyName = "virtualTouchID",
 	virtualEventNames = "vmouseover vmousedown vmousemove vmouseup vclick vmouseout vmousecancel".split( " " ),
+	generalProps = (
```

Comment:
These are the "stock" properties that jQuery 1.x/2.x copies to an event. Both this code and the original code are broken in that they will miss any properties that others add to the core list after this code runs, but if nobody has reported it then I guess you'd wait until it became a problem.